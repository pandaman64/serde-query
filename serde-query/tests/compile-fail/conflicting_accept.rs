use serde_query::Deserialize;

#[derive(Debug, Deserialize)]
struct Data {
    #[query(".foo.bar")]
    bar1: String,
    #[query(".foo.bar")]
    bar2: String,
}

fn main() {}
