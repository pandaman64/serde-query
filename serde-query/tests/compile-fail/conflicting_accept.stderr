error: Cannot use the same query for two or more fields: 'bar1', 'bar2'
 --> tests/compile-fail/conflicting_accept.rs:6:5
  |
6 |     bar1: String,
  |     ^^^^
